packages/validators-util/package.json

```
@@ -0,0 +1,45 @@
+{
+    "name": "@fremtind/jkl-validators-util",
+    "version": "1.0.0",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "Simple validation functions to use for validating",
+    "homepage": "https://fremtind.github.io/jokul",
+    "keywords": [
+        "validation",
+        "utils",
+        "jøkul"
+    ],
+    "license": "MIT",
+    "types": "./build/index.d.ts",
+    "main": "./build/cjs/index.js",
+    "module": "./build/esm/index.js",
+    "browser": "./build/browser/index.js",
+    "directories": {
+        "lib": "build"
+    },
+    "files": [
+        "build"
+    ],
+    "scripts": {
+        "clean": "rimraf node_modules/ .cache/ build/ dist/ public/ **/*.css",
+        "build:types": "tsc -p tsconfig-for-declarations.json",
+        "build:scripts": "rollup --config ../../rollup.config.js",
+        "build": "run-s build:*",
+        "test": "echo \"Error: run tests from root\" && exit 1",
+        "dev:style": "echo noop"
```

Mangler et komma her
```suggestion
        "dev:style": "echo noop",
```

---

Kanskje like greit å bare slette den, det burde ikke ha noe å si for bygget.